src/main/java/org/spongepowered/api/event/player/PlayerInteractEvent.java

```diff
@@ -37,11 +36,11 @@
 public interface PlayerInteractEvent extends PlayerEvent, Cancellable {
 
     /**
-     * Gets the {@link BlockLoc} that the player has clicked, if available.
+     * Gets the {@link BlockLoc} that the player has clicked.
      * 
      * @return The block
      */
-    Optional<BlockLoc> getBlock();
+    BlockLoc getBlock();
```

Comment:
Please comment on things that are more up to date. This issue does not exist on [master](https://github.com/SpongePowered/SpongeAPI/blob/master/src/main/java/org/spongepowered/api/event/player/PlayerInteractEvent.java#L35).